### PR TITLE
update homepage icons

### PR DESCRIPTION
**Pull request by Deflaimun**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2485
Review deadline:

## Page previews

https://deploy-preview-525--redpanda-docs-preview.netlify.app/current/home/

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Comment by Deflaimun** [comment]
Waiting for 
- [x] Image in PNG format
- [x] Link for the new page

Edit. Done!

File: antora.yml [hunk 1]
```diff
@@ -37,10 +37,10 @@ asciidoc:
...
         url: 'deploy:deployment-option/cloud/serverless.adoc'
         learn-text: true
-      - title: 'Redpanda Cloud'
-        image: cloud-first.png
-        description: 'A complete data streaming platform delivered as a fully-managed service.'
-        url: 'deploy:deployment-option/cloud/cloud-overview.adoc'
+      - title: 'Redpanda Connect'
+        image: redpanda-connect.png
+        description: 'Redpanda Connect provides an ecosystem of 220+ open source connectors.'
+        url: 'redpanda-connect:ROOT:index.adoc'
```
Comment (Deflaimun) [inline antora.yml]: @JakeSCahill is that the expected URL?

**Comment by Feediver1** [comment]
So once you have the .png, the icon will render correctly--right now:
image

File: antora.yml (hunk 1 above)
Comment (Deflaimun) [inline antora.yml]:
```suggestion
        url: 'redpanda-connect:ROOT:about.adoc'
```

**Comment by Deflaimun** [comment]
@JakeSCahill feel free to merge whenever

**Review by Feediver1 — APPROVED** [review]
Please confirm the link provided by @JakeSCahill works as expected. Thanks!

**Pull request merged**